src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,23 @@
 package core.basesyntax;
 
+import core.basesyntax.model.Bulldozer;
+import core.basesyntax.model.Excavator;
+import core.basesyntax.model.Machine;
+import core.basesyntax.model.Truck;
+
 public class MainApp {
+    public static void main(String[] args) {
+        doAndStopWorkForEachMachine();
+    }
+
+    private static void doAndStopWorkForEachMachine() {
+        for (Machine machine : buildMachines()) {
```

Comment:
I think result of buildMachines() method should be extracted to variable, and then used in loop